coucharchive

```diff
@@ -493,12 +504,13 @@ def main():
     if args.action == 'create':
         create(args.source_server, args.output, ignore_dbs=ignore_dbs)
     elif args.action == 'restore':
-        restore(args.target_server, args.input, ignore_dbs=ignore_dbs)
+        restore(args.target_server, args.input,
+                args.force_continue, ignore_dbs=ignore_dbs)
```

Comment:
same here, if argument is optional, passing it explicitly will reduce the risk of bugs